include/tvm/runtime/c_backend_api.h

```diff
@@ -152,6 +152,16 @@ TVM_DLL int TVMBackendParallelBarrier(int task_id, TVMParallelGroupEnv* penv);
  */
 TVM_DLL int TVMBackendRunOnce(void** handle, int (*f)(void*), void* cdata, int nbytes);
 
+/*!
+ * \brief Reset the threads in the pool. All current threads are destroyed and
+ * new ones are created.
+ *
+ * Note that this does nothing when openmp is used.
+ *
+ * \return 0 when no error is thrown, -1 when failure happens
+ */
```

Comment:
Unless needed from the backend generated code, we should avoid adding new functions through C API, as C API needs to remain generally stable. Instead, consider register a packed function to do so